pkg/container-collection/containers.go

```diff
@@ -22,15 +22,20 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/dynamic"
 	"k8s.io/client-go/rest"
+
+	"github.com/kinvolk/inspektor-gadget/pkg/columns"
 )
 
 // Container represents a container with its metadata.
 type Container struct {
+	// Container Runtime
+	Runtime string `json:"runtime,omitempty" column:"runtime,width:10,fixed" columnTags:"runtime"`
```

Comment:
Once min/max feature is ready, it will become min=5 (cri-o) and max=10 (containerd).

Reply:
Done.